### DISCUSSION TITLE
How to prevent '&' from being converted to '&amp;' in a URL?

### DISCUSSION THREAD

**Discussion by mingming-ma** [remarkjs/remark#1286] [Q&A] [answered]
Here is the screen shot that shows the converted url when cursor on the rendered text. You can find the & is converted to "\&amp;" 

<img width="1279" alt="image" src="https://github.com/remarkjs/remark/assets/133393905/f0f912f2-ed16-4cf7-b472-c42a02892e35">

I would like to keep & in the url so user can watch the Youtube from a start time. Any suggestions to fix this?

**Comment by wooorm** [comment, marked as answer]
run `npm update`

**Reply by ChristianMurphy** [reply]
@wooorm does the https://remarkjs.github.io/react-markdown/ site need to be rebuilt with the latest version of `react-markdown` so the fix appears there too?

**Reply by wooorm** [reply]
Ah, that can definitely be. A push on the website branch would do the trick

**Reply by ChristianMurphy** [reply]
Opened https://github.com/remarkjs/react-markdown/pull/814 to resolve this on the site

**Comment by mingming-ma** [comment]
Appreciate the quick response! Thank you!
